### ISSUE TITLE
CentOS intall process should have install.sh wrapper script

### ISSUE THREAD

**Issue by cmosetick** [RevolutionAnalytics/RRO#53]
Ubuntu installers have a `install.sh` wrapper script. CentOS should have the same to simplify the install instructions across all platforms.

e.g.
```
#!/bin/bash
# install.sh wrapper
yum --nogpgcheck localinstall RRO-8.0.1-Beta-el6.x86_64.rpm
```

**Comment by revodavid** [comment]
@sfweller can you confirm fixed for 8.0.3?

**Comment by sfweller** [comment]
Confirmed that 'install.sh' script has been updated as planned for 8.0.3 release.
